cmd/kube-apiserver/app/apiextensions.go

```diff
@@ -39,7 +39,7 @@ import (
 func createAPIExtensionsConfig(
 	kubeAPIServerConfig genericapiserver.Config,
 	externalInformers kubeexternalinformers.SharedInformerFactory,
-	pluginInitializers []admission.PluginInitializer,
+	_ []admission.PluginInitializer,
```

Comment:
it's not very common to replace unused params with `_` in this repo... unless there's a specific reason we want to highlight this is unused, I would recommend reverting those changes